public/src/components/channelManagement/epicTests/epicVariantPreview.tsx

```diff
@@ -125,4 +135,40 @@ const EpicVariantPreview: React.FC<EpicVariantPreviewProps> = ({
   return <div className={classes.container}>{Epic && <Epic {...props} />}</div>;
 };
 
+/**
+ * Uses the DCR storybook to render the component, iframed.
+ * Props are passed in the args parameter in the url.
+ */
+const EpicVariantPreviewStorybook = ({ variant, moduleName }: EpicVariantPreviewProps) => {
+  const classes = useStyles();
+
+  const tickerSettingsWithData = useTickerData(variant.tickerSettings);
+  const props = buildProps(variant, tickerSettingsWithData);
+  const compressedProps = lzstring.compressToEncodedURIComponent(JSON.stringify(props));
+
+  const storyName = StorybookNames[moduleName];
+  const dcrStorybookUrl = 'https://5dfcbf3012392c0020e7140b-borimwnbdl.chromatic.com';
+
+  return (
+    <div>
+      <iframe
+        className={classes.iframe}
+        src={`${dcrStorybookUrl}/iframe.html?id=${storyName}&viewMode=story&shortcuts=false&singleStory=true&args=json:${compressedProps}`}
```

Comment:
Has the PR which adds support for reading these compressed args at the other end been merged?

Reply:
Not yet! It's part of the same PR that adds the component to DCR. I'll merge that one first